### ISSUE TITLE
_api_slug is not flexible enough

### ISSUE THREAD

**Issue by srallen** [zooniverse/panoptes-python-client#18]
@astopy When working with the team member from NU yesterday, we determined that we should add a couple more custom `find()` calls into the client that go to the `classifications/project` endpoint and the `set_member_subjects` endpoint. The first would be so that all classifications for the Gravity Project can be polled for. The other is to find the subject set ids that belong to a subject.

I'd prefer not to create two new class objects just for these, but instead incorporate these into the existing `Subject` and `Classification` objects. For instance, I'd like to have a method for `Subject.subject_sets('subject_id'='342')` that returns the 'set_member_subjects' endpoint. 

`_api_slug` is used throughout the PanoptesObject and integral for building the appropriate api calls. I could modify this as needed, but this may have some unintended issues. How would you suggest going forward?

**Comment by astopy** [comment]
OK, so finding the classifications for a project is already possible with:

```python
Classification.where(project_id=2731)
```

(Note that `.find()` is for looking up one specific object, while `.where()` is for querying for multiple objects.)

But allowing querying scopes will be useful, so see #21 for that anyway.

I'm still looking into supporting `set_member_subjects`.

**Comment by astopy** [comment]
OK, I think it does just make sense to add a `SetMemberSubject` class. See #22.

**Comment by srallen** [comment]
About the classifications endpoint, this confused me too for a while, but `/classifications` with a `project_id` query param will return the classifications _you_ have made for that project, unless you are authenticated as an admin. We need to fetch _all_ classifications for a project, which is what `/classifications/project` is for. I'll try out #21 and see if it returns what we need and take a look at #22 too. Thanks.

**Comment by astopy** [comment]
Ah I see. The docs aren't very clear on that :)

```python
Classification.where(project_id=2731, scope='project')
```

... in #21 should do it then.

**Comment by srallen** [comment]
Merged #21 and #22. Closing.
